src/core/src/adapter.rs

```diff
@@ -5,6 +5,9 @@
 use {Backend, Gpu, QueueType};
 
 /// Represents a physical or virtual device, which is capable of running the backend.
+///
+/// The `Adapter` is typically obtained from objects implementing `gfx::WindowExt`. Different 
```

Comment:
We also have [`Headless`](https://github.com/gfx-rs/gfx/blob/master/src/core/src/lib.rs#L346-L353) now. The difference between `WindowExt` and `Headless` is the additional `Surface` as we basically have 2 entrypoints for the library `Adapter` for the physical GPU side and `Surface` for the physical display side. Both open/create logical equivalents on the library side (`Gpu` and `Swapchain`) for working with the API.